src/SFA.DAS.LevyTransferMatching.Api/Controllers/PledgeController.cs

```diff
@@ -144,6 +145,22 @@ public async Task<IActionResult> ClosePledge(int pledgeId, [FromBody] SetClosePl
             }
         }
 
+        [HttpPost]
+        [Route("accounts/{accountId}/pledges/{pledgeId}/rejectapplications")]
```

Comment:
Nit-picky, but we should hyphenate the route for readability.

Reply:
I have updated the route now